jupiterone/questions/questions.yaml

```diff
@@ -1,15 +1,20 @@
 ---
```

Comment:
spelling need to be updated to "HIPAA" in all instances

Reply:
"mimecast" should be capitalized in all instances

Reply:
George's made the changes although I'm uncertain why the PR validate target is failing.